sentry/examples/anyhow-demo.rs

```diff
@@ -0,0 +1,19 @@
+fn execute() -> anyhow::Result<usize> {
+    let parsed = "NaN".parse()?;
+    Ok(parsed)
+}
+
+fn main() {
+    let _sentry = sentry::init((
+        "https://a94ae32be2584e0bbd7a4cbb95971fee@sentry.io/1041156",
```

Comment:
Which DSN is this?

Reply:
Let's actually back out all DSNs. We could configure via env_var if we want to test this, or just put a placeholder in.

Reply:
Let me remove that from all other examples as a followup.